### PR TITLE
Killjuicy challenge

### PR DESCRIPTION
**Pull request by Scar26**

### CODE REVIEW

File: data/static/challenges.yml [hunk 1]
```diff
@@ -1084,6 +1084,14 @@
   mitigationUrl: 'https://cheatsheetseries.owasp.org/cheatsheets/Choosing_and_Using_Security_Questions_Cheat_Sheet.html'
   key: geoStalkingVisualChallenge
 -
+  name: 'Kill Juicy'
```
Comment (bkimminich) [inline data/static/challenges.yml]: The challenge name should be neutral b/c the bot name is configurable. Like "Kill Chat Bot" for example?

Comment (Scar26) [reply data/static/challenges.yml]: updated

File: routes/chatbot.js [hunk 2]
```diff
@@ -47,8 +48,24 @@ async function processQuery (user, req, res) {
...
+    res.status(200).json(response)
+  } catch (err) {
+    try {
+      await bot.respond(testCommand, user.id)
+      res.status(200).json({
+        action: 'namequery',
+        body: 'I\'m sorry I didn\'t get your name. What shall I call you?'
+      })
+    } catch (err) {
+      utils.solve(challenges.killJuicyChallenge)
```
Comment (bkimminich) [inline routes/chatbot.js]: Either check if the challenge was solved before running `utils.solve` or use `utils.solveIf` and pass the condition in as a callback function.

File: data/static/challenges.yml [hunk 3]
```diff
@@ -1083,6 +1083,15 @@
   hintUrl: 'https://pwning.owasp-juice.shop/part2/sensitive-data-exposure.html#determine-the-answer-to-emmas-security-question'
   mitigationUrl: 'https://cheatsheetseries.owasp.org/cheatsheets/Choosing_and_Using_Security_Questions_Cheat_Sheet.html'
   key: geoStalkingVisualChallenge
+-
+  name: 'Kill Chatbot'
+  category: 'Improper Input Validation'
+  description: 'Permanently disable the Juicy Chatbot so that it can no longer answer customer queries'
```
Comment (bkimminich) [inline data/static/challenges.yml]: Rephrase into "disable the support chatbot" to remove direct "juice" reference for custom themes.

**Pull request merged**